src/generate_conf.rs

```diff
@@ -269,7 +329,13 @@ mod tests {
         "#,
         )?;
 
+        let config_files = vec![
+            generate_config_file("eth1".to_string(), "eth1".to_string()),
+            generate_config_file("bridge0".to_string(), "bridge0".to_string()),
+        ];
+
         let mut interfaces = extract_interfaces(&net_state);
+        populate_connection_ids(&mut interfaces, &config_files).expect("populate ids");
```

Comment:
I believe we're fine to drop this. The test explicitly wants to verify that `extract_interfaces` works as expected, not necessarily the complete state of the final output.

Reply:
True, however than in the assert, we've to specify empty vectors in connection_ids. Which is confusing from a developers perspective